composer.json

```
@@ -136,6 +141,7 @@
     "scripts-descriptions": {
         "drupal:install": "Installs Starshot.",
         "drupal:install-dev": "Installs Starshot, with additional modules and configuration tweaks for development.",
+        "drupal:rebuild": "Rebuilds the codebase and reinstalls Starshot from scratch.",
```

Let's mention that this should only be used for internal development...

```suggestion
        "drupal:rebuild": "Rebuilds the codebase and reinstalls Starshot from scratch. Should only be used for internal development.",
```